test/serviceBusManagementClientTests.ts

```diff
@@ -0,0 +1,241 @@
+import { SasTokenProvider, TokenInfo } from "@azure/amqp-common";
```

Comment:
Can you suffix the test files with spec.ts. I learnt about this new convention and like it.